yarn.lock

```diff
@@ -11528,6 +11528,53 @@ react-is@^18.0.0, react-is@^18.2.0:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-18.2.0.tgz#199431eeaaa2e09f86427efbb4f1473edb47609b"
   integrity sha512-xWGDIW6x921xtzPkhiULtthJHoJvBbF3q26fzloPCK0hsvxtPVelvftw3zjbHWSkR2km9Z+4uxbDDK/6Zw9B8w==
 
+react-native-windows@0.73.0-preview.7:
```

Comment:
this shouldn't be here - every package should be updated to use react-native-windows@0.73.0 so this isn't present.